arch/xtensa/core/startup/reset-vector.S

```diff
@@ -433,10 +433,16 @@ _xtos_mpu_attribs:
 #elif XCHAL_HAVE_CACHEATTR || XCHAL_HAVE_MIMIC_CACHEATTR \
 		|| XCHAL_HAVE_XLT_CACHEATTR \
 		|| (XCHAL_HAVE_PTP_MMU && XCHAL_HAVE_SPANNING_WAY)
+/* FixMe: Setting CACHEATTR from _memmap_cacheattr_reset is making the
+ * Intel_S1000 SoC get into some unknown state. Removing them only for
```

Comment:
Dumb question: are you sure you got the value of memmap_cacheattr_reset right in the linker script?  That seems like a weird place to have to store it, and it's reliant on developers typing them correctly instead of emitting it as part of the XCHAL_* stuff.